dell/xps/15-7590/nvidia/default.nix

```
@@ -21,5 +21,7 @@
       # Bus ID of the NVIDIA GPU.
       nvidiaBusId = lib.mkDefault "PCI:1:0:0";
     };
+
+    open = lib.mkDefault true;
```

To avoid the conflicting option:

```suggestion
    open = true;
```